src/pages/stop-details/stop-details.html

```diff
@@ -20,20 +20,30 @@
       <ion-title align="center">{{this.stopName}}</ion-title>
 
       <ion-buttons end>
+        <!-- <<<<<<< HEAD -->
 
-      <button ion-button clear icon-only class="googleMapPin" (click)="MapbuttonClick()">
+        <button ion-button clear icon-only class="googleMapPin" (click)="MapbuttonClick()">
 
+<!-- ======= -->
```

Comment:
why are these comments left in? I think we should remove all of the comments in this file